stubs/pyinstaller/PyInstaller/utils/hooks/conda.pyi

```diff
@@ -0,0 +1,53 @@
+# https://pyinstaller.org/en/stable/hooks.html?highlight=conda_support#module-PyInstaller.utils.hooks.conda
+
+import sys
+from _typeshed import StrOrBytesPath
+from collections.abc import Sequence
+from pathlib import Path, PurePath
+from typing import Any
+from typing_extensions import TypeAlias, TypedDict
+
+if sys.version_info >= (3, 8):
+    from importlib.metadata import PackagePath as _PackagePath
+else:
+    _PackagePath: TypeAlias = Any
+
+CONDA_ROOT: Path
+CONDA_META_DIR: Path
+PYTHONPATH_PREFIXES: list[Path]
+
+class _RawDict(TypedDict):
+    name: str
+    version: str
+    files: list[StrOrBytesPath | PurePath]
```

Comment:
`PurePath` is redundant in this union, since `PurePath` inherits from `PathLike[str]`, and `StrOrBytesPath` is just `str | bytes | PathLike[str] | PathLike[bytes]`

```suggestion
    files: list[StrOrBytesPath]
```